### DISCUSSION TITLE
大佬想问一下你的滴答清单任务管理系统大概是什么粒度

### DISCUSSION THREAD

**Discussion by Ffffffffchopin** [oldwinter/knowledge-garden#54] [提问]
您提到每日计划不应该细到小时级，否则就成打卡了，但是在《我的日程》中也提到日程安排会有控制感，Todo List有必要，想知道您每日的任务管理工作流是什么样的呢，任务管理是更接近目标设定还是日程表呢

**Comment by oldwinter** [comment]
主要当成待办和备忘录，更接近日程表吧，或者说就是一条条简单的清单。目标设定这些，生活部分，我也会简单在滴答里写一下，但也主要当成备忘录用了，比如目标今年减20斤这种。工作上的目标，则是需要和同事们配合，用公司规定的协同工具，这类工具，我觉得notion这种database是很合适的，当然大公司一般也都有自研的内部管理工具。
